Models/Entities/ParkedVehicle.cs

```diff
@@ -5,15 +5,27 @@ namespace Garage2._0.Models.Entities
     public class ParkedVehicle
     {
         public int Id { get; set; }
-        public string VehicleType { get; set; }
+
+        [Required(ErrorMessage = "Vehicle type is required.")]
+        public VehicleType VehicleType { get; set; }
+
+        [Required]
+        [MaxLength(10)]
+        [RegularExpression("^[A-Z0-9]*$", ErrorMessage = "Registration number should be uppercase alphanumeric.")]
```

Comment:
instead of forcing the user to write it in uppercase, shouldn't we do it inside a method? To convert the whole input to uppercase, or rather, while filing the form, all letters that are input in the form they're automatically uppercase?